recipes/libmodulemd/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "libmodulemd" %}
+{% set version = "2.9.4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/fedora-modularity/libmodulemd/releases/download/libmodulemd-{{ version }}/modulemd-{{ version }}.tar.xz
+  sha256: cb86b1dad4f1578895225ba4ee435dbb7d75262898f69a08507b01759bfc81ab
+
+build:
+  number: 0
+  skip: true  # [win or osx]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - pkg-config
+    - ninja
+  host:
+    - python
+    - file
+    - glib
+    - yaml
+    - rpm-tools
+    - gobject-introspection
+    - pygobject
+    - meson
+    - help2man
+  run:
+    - python
+    # https://abi-laboratory.pro/index.php?view=timeline&l=rpm
+    - {{ pin_compatible('rpm-tools', max_pin="x.x") }}
+    - {{ pin_compatible('file') }}
+
+test:
+  commands:
+    - modulemd-validator --help
+  imports:
+    - gi
```

Comment:
Doesn't this test that `gobject-introspection` has made it to `run` only? (Maybe that's intentional.)